productPage/productPage.html

```diff
@@ -0,0 +1,616 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
```

Comment:
Not mandatory but you can also use Meta tags for author and description here for best practice

Reply:
not so used to it but i ll try